draft getting started (#12392)
PostHog/posthog.com: contents/docs/llm-analytics/_snippets/span-event.mdx

> A span is a single action within your application i.e. a function call or vector database search.

Reviewer: ```suggestion
A span is a single action within your application, such as a function call or vector database search.
```
